FAQs/README.md

```diff
@@ -114,6 +116,25 @@ Using the native enterprise directory for authentication ensures that there is a
 <br>
 <br>
 
+### **7. How can I migrate classic Application Insights to Workspace-based? If I migrate Application Insights AzSK-AzTS-AppInsights-xxxxx to Workspace-based, will there be any impact to AzTS scan?**
+It is recommended to migrate to classic Application Insights to Workspace-based before retirement date(29th February, 2024) to ensure most reliable experience. You can follow steps mentioned in detail [here](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#migrate-your-resource) for migration using any one of the below listed approaches.
+
+> [Azure Portal](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#migrate-your-resource)
+<br/>
+
+> [Azure CLI](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#azure-cli)
+<br/>
+
+> [Azure PowerShell](https://learn.microsoft.com/en-us/azure/azure-monitor/app/convert-classic-resource#azure-powershell)
+<br/>
+
+You can use either Workspace in AzTS host Resource group (AzSK-AzTS-LAWorkspace-xxxxx) or any other Workspace (new or existing). This migration will not impact AzTS scan in any way.
+
+We have also updated our deployment package to have this change incorporated. Any new setups/ rerun of installtion script using latest deployment package will result in creation or updation of Application Insights to Workspace-based mode. 
+
+As per latest communication from Azure, for Application Insights not migrated to Workspace-based by retirement date, ingestion will continue until it is auto-migrated.AzTS scan is expected to be non impacted in such cases.  However, for reliable experience, it is recommended to switch to Workspace-based mode by 29th February, 2024.
```

Comment:
AzTS scan is expected to be non impacted in such cases - > AzTS scan will not be impacted due to this.